<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-2K9I04N" failures="0" tests="1" name="Default test" time="0.069" errors="0" timestamp="2023-09-25T15:36:16 IST">
  <testcase classname="Base.BaseClass" name="@BeforeSuite createSuite" time="1.8">
    <failure type="com.gurock.testrail.APIException" message="TestRail API returned HTTP 400(&amp;quot;Field :suite_id is a required field.&amp;quot;)">
      <![CDATA[com.gurock.testrail.APIException: TestRail API returned HTTP 400("Field :suite_id is a required field.")
at com.gurock.testrail.APIClient.sendRequest(APIClient.java:296)
at com.gurock.testrail.APIClient.sendPost(APIClient.java:132)
at Base.BaseClass.createSuite(BaseClass.java:59)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:390)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:325)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:382)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeSuite createSuite -->
  <testcase classname="Base.BaseClass" name="@BeforeTest LaunchApplication" time="0.0">
    <skipped/>
  </testcase> <!-- @BeforeTest LaunchApplication -->
  <testcase classname="Base.BaseClass" name="@BeforeMethod beforeTest" time="0.0">
    <skipped/>
  </testcase> <!-- @BeforeMethod beforeTest -->
  <testcase classname="Base.BaseClass" name="@AfterMethod afterTest" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterMethod afterTest -->
  <testcase classname="Base.BaseClass" name="@AfterTest TearDown" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterTest TearDown -->
  <testcase classname="TestRail_TestNG.FaceBook_TestNG" name="Title_Check" time="0.0">
    <skipped/>
  </testcase> <!-- Title_Check -->
</testsuite> <!-- Default test -->
